Cargo.toml

```diff
@@ -17,6 +17,9 @@ license = "MIT"
 [features]
 unstable = []
 
+[dependencies]
+slotmap = "1.0.6"
```

Comment:
the other dependencies are formatted like `[dependencies.crate]`, but now that `cargo add <crate>` exists, this is the default form, unless a crate gets more advanced features set